jenkins/job-configs/kubernetes-jenkins/kubernetes-djmm.yaml

```diff
@@ -47,16 +47,17 @@
             timeout: 50
             job-env: |
               export RELEASE_INFRA_PUSH=true
+              export SET_NOMOCK_FLAG=false
         - 'federation-build':
             branch: 'master'
             timeout: 50
             job-env: |
               export RELEASE_INFRA_PUSH=true
+              export SET_NOMOCK_FLAG=false
               export PROJECT="k8s-jkns-e2e-gce-federation"
               export FEDERATION=true
-              export FEDERATION_PUSH_REPO_BASE="gcr.io/k8s-jkns-e2e-gce-federation"
+              export FEDERATION_PUSH_REPO_BASE="gcr.io/djmm-federation-build"
```

Comment:
I suggested this assuming that the PROJECT in your job is also same.
Is this the project in which you want to run your job? (Make sure that it has right service accounts)
If yes, then set the PROJECT to "djmm-federation-build" as well.